src/helpers/class-components.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Helpers for components
+ *
+ * @package Eightshift_Libs\Helpers
+ */
+
+declare( strict_types=1 );
+
+namespace Eightshift_Libs\Helpers;
+
+/**
+ * Helpers for components
+ */
+class Components {
+
+  /**
+   * Makes sure the output is string. Useful for converting an array of components into a string.
+   *
+   * @param  array|string $array_or_string Variable we need to convert into a string.
+   * @return string
+   */
+  public static function ensure_string( $array_or_string ) : string {
```

Comment:
```suggestion
  public static function ensure_string( $variable ) : string {
```

Just reads weird when I see `$array_or_string`, there is already a docblock that specifies `array|string`. Don't think we need to be so verbose 😄

Reply:
What can I say, I'm a sucker for verbosity. Will change. 👍